compiler/rustc_codegen_cranelift/Cargo.toml

```diff
@@ -23,7 +23,8 @@ libloading = { version = "0.8.0", optional = true }
 smallvec = "1.8.1"
 
 [patch.crates-io]
-# Uncomment to use local checkout of cranelift
+# todo: remove patch
+compiler_builtins = { git = "https://github.com/tgross35/compiler-builtins.git", package = "compiler_builtins", branch = "overflowing-c-safe-ret" }
```

Comment:
This doesn't apply to building the sysroot. Also the patch in library/Cargo.toml should be enough to apply while building a sysroot for cg_clif.